contactsummary/scss/contactsummary.scss

```diff
@@ -1,4 +1,5 @@
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
+@import "SCSSROOT/org.civicrm.shoreditch/scss/civicrm/variables";
```

Comment:
why is this necessary? nothing in the civicrm side of Shoreditch should be used by apps that use bootstrap-based markup